src/tools/CBExport.ts

```diff
@@ -0,0 +1,143 @@
+        if(!connection){
+            return;
+        }
+
+        const allScopes = scopes.includes("All Scopes");
+        const scp: string[] = [];
+        const collections: string[] = [];
+
+        if (!allScopes) {
+            for (const col of cols) {
+                if (col.includes(" ")) { // All XXX (XXX represents scopeName)
+                    scp.push(col.split(" ")[1]);
+                }
+            }
+
+            for (const col of cols) {
+                if (!col.includes(" ") && !CBExport.hasAnyScope(col, scp)) { // TODO: Improve hasAnyScope
+                    collections.push(col);
+                }
+            }
+        }
+
+        scp.push(...collections);
+
+        const fileName = `${bucket}_cbexport_${getCurrentDateTime()}.json`;
+        let currentPath: string;
+
+        if (filePath.endsWith("/")) { //TODO: Verify for windows
+            currentPath = filePath + fileName;
+        } else {
+            currentPath = `${filePath}/${fileName}`;
+        }
+
+        const includeData = scp.join(",");
+        const fullPath = currentPath;
+
+        // const task = new ProgressIndicator.Task.Backgroundable(
+        //     null,
+        //     `Exporting '${bucket}'`,
+        //     true
+        // );
+
+        // task.run((indicator: ProgressIndicator) => {
+        //     indicator.setIndeterminate(true);
+
+        const password = await keytar.getPassword(Constants.extensionID, getConnectionId(connection));
+        if (!password) {
+            return undefined;
+        }
+
+        try {
+            
+            const cmd: string[] = [];
+            cmd.push(CBTools.getTool(Type.CB_EXPORT).path);
+            cmd.push("json");
+            cmd.push("--no-ssl-verify");
+            cmd.push("-c");
+            cmd.push(connection.url); // Get Cluster URL
+            cmd.push("-u");
+            cmd.push(connection.username); //Get Username
+            cmd.push("-p");
+            cmd.push(password); // get Password
+            cmd.push("-b");
+            cmd.push(bucket);
+
+            if (!allScopes) {
+                cmd.push("--include-data");
+                cmd.push(includeData);
+            }
+
+            cmd.push("--scope-field");
+            cmd.push(scopeName);
+            cmd.push("--collection-field");
+            cmd.push(colName);
+            cmd.push("--include-key");
+            cmd.push(keyName);
+            cmd.push("-f");
+            cmd.push(format);
+            cmd.push("-o");
+            cmd.push(fullPath);
+            cmd.push("-t");
+            cmd.push(threads);
+
+            if (verbose) {
+                cmd.push("-v");
+            }
+
+            const terminal = vscode.window.createTerminal("CBExport",);
+            let text = cmd.join(" ");
+            terminal.sendText(text);
+            terminal.show();
+
+            // const process = child_process.spawnSync(cmd[0], cmd.slice(1), {
```

Comment:
Can you please remove the commented code?

Reply:
done